cms_frontend/src/components/FileRenderer/FileRenderer.tsx

```diff
@@ -42,34 +45,37 @@ const sortFiles = (files: FileFormat[]) => {
 // Typescript Declaration
 // uses the interface defined above
 // imports the icons from material UI
-const FileRenderer: React.FC<RenderProps> = ({ files, onFileClick, onFolderClick, onNewFile }) => {
+const FileRenderer: React.FC<RenderProps> = (props) => {
+  const { files, onFileClick, onFolderClick, onRename, onNewFile } = props;

```

Comment:
lets keep the style to be consistenly ({props}) instead of destructuring it afterwards.